### ISSUE TITLE
Dapr dashboard not found (Windows, dapr v1.0)

### ISSUE THREAD

**Issue by maiorfi** [dapr/dashboard#130]
## Expected Behavior

Dashboard starts listening on localhost:8080

## Actual Behavior

```powershell
PS C:\Users\userhome> dapr dashboard
Dapr Dashboard running on http://localhost:8080
Dapr dashboard not found. Is Dapr installed?
```

## Steps to Reproduce the Problem

On windows 10, just run "dapr dashboard".

**Comment by maiorfi** [comment]
On same machine, installing dapr-cli and initializing dapr on WSL2 dashboard works as expected, despite I cannot use some ports, such as default one (8080) without any evident reason. Starting it for example using port 777 (i.e. running in wsl2 bash shell "dapr dashboard -p 7777") works fine.

By the way, although docker containers are shared between windows os and wsl2 (docker ps shows same containers, since docker desktop is configured to use wsl2 host), running "dapr init" on one side and "dapr list" on the other does not work, i.e. one side cannot "see" dapr instance initialized on the other one.

**Comment by artursouza** [comment]
@maiorfi maybe port 8080 is being used in another process.

Now, regarding the cross windows and wsl2 list issue, please cut an issue in cli for that. The good news is that we know why it is going on. The bad news is that the fix might not be that easy/worth. We use the list of running cli processes to discover dapr runs - we previously had a shared master file but that would become stale as processes got forcefully killed and the master list did not get updated. So, the authoritative source of running processes remains the OS.

**Comment by maiorfi** [comment]
Sadly, "dapr dashboard -p xxxx" doesn't work for any port I choose when run in powershell (neither using admin privileges), while it works fine in WSL2.

I guess it's a "doesn't work on my machine" problem, right?

Concerning cross win/wsl list, well, it's not an issue by itself, just a noteworthy behavior :)

**Comment by maiorfi** [comment]
Issue was (apparently) solved by dapr version 1.1

**Comment by artursouza** [comment]
Good to know! Thanks for the update.
